nifi-api/src/test/resources/TestVariableRegistry/foobar.properties

```diff
@@ -0,0 +1 @@
+fake.property.3=test me out 3, test me out 4
```

Comment:
@YolandaMDavis I would prefer adding in the ASF license. RAT check should be sort of a last resort, if we cannot use the license - for example, a binary file or a file that doesn't allow comments, etc.